sdk/benches/has_duplicates.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(test)]
+
+extern crate test;
+use solana_sdk::sanitized_transaction::SanitizedTransaction;
+use test::Bencher;
+
+#[bench]
+fn bench_has_duplicates(bencher: &mut Bencher) {
+    bencher.iter(|| {
+        let data = test::black_box([1, 2, 3]);
```

Comment:
nitpicking: perhaps a more realistic array size would provide better benchmark.

Reply:
This is just a move of a pre-existing benchmark. I do agree though and would like to make duplicate detection more efficient for larger account lists made possible by the transaction v2 proposal.